src/reuse/_comment.py

```diff
@@ -678,6 +688,7 @@ class UncommentableCommentStyle(EmptyCommentStyle):
     "ROOT": MlCommentStyle,
     "setup.cfg": PythonCommentStyle,
     "sonar-project.properties": PythonCommentStyle,
+    ".vimrc": VimCommentStyle,
```

Comment:
Nitpick: the sorting doesn't look right. The dotfiles are at the beginning of the list.

Reply:
Sure thing :+1: